src/main/java/com/jagrosh/jmusicbot/commands/music/QueueCmd.java

```diff
@@ -86,6 +90,32 @@ public void doCommand(CommandEvent event)
             });
             return;
         }
+
+        if(pagenum > ah.getQueue().getNumberOfPages()){
+            event.reply("That page does not exist. The maximum number of pages on the queue is " + String.valueOf(ah.getQueue().getNumberOfPages()) + ". Showing last page.");
```

Comment:
We shouldn't be sending an extraneous message when we're not changing the behavior _and_ the changed behavior is already visible (in the different page number).